e2e/scenario_test.go

```diff
@@ -559,6 +559,20 @@ func Test_ubuntu1804ChronyRestarts(t *testing.T) {
 	})
 }
 
+func Test_ubuntu2204SelfContainedInstaller(t *testing.T) {
+	RunScenario(t, &Scenario{
+		Description: "tests that a new ubuntu 2204 node using self contained installer can be properly bootstrapped",
+		Config: Config{
+			NodeBootstrappingType: SelfContained,
```

Comment:
do we want to use that type `selfcontained` ?  I know we moved to calling this `scriptless` movgin forward. 

What are other expected types ?

Reply:
```
type NodeBootstrappingType string

const (
	CustomScripts NodeBootstrappingType = "CustomScripts"
	Scriptless    NodeBootstrappingType = "Scriptless"
)
```